plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -681,16 +606,6 @@ private fun PackageJson.getScopeDependencies(type: YarnDependencyType) =
         YarnDependencyType.DEV_DEPENDENCIES -> devDependencies
     }
 
-/**
- * Check whether Corepack is enabled based on the `package.json` file in [workingDir]. If no such file is found
- * or if it cannot be read, assume that this is not the case.
- */
-private fun isCorepackEnabledInManifest(workingDir: File): Boolean =
```

Comment:
The commit message should briefly mention that `isCorepackEnabledInManifest` is also moved, while at it, as `Yarn2Command` is the only caller.